### DISCUSSION TITLE
Updating bropats libraries

### DISCUSSION THREAD

**Discussion by RetroVape** [fuatakgun/eufy_security#500] [Q&A]
Bropats libraries have been updated with a ton of fixes. Is there a way to drop in the new libraries on the HA side, or does this project need to be rebuilt with them?

**Comment by fuatakgun** [comment]
this project needs to be re-tested with them and I am trying to do my best to catch up his speed

**Reply by RetroVape** [reply]
I get it. I really appreciate your efforts!

**Comment by fliespl** [comment]
@fuatakgun is it possibile to install newer bropat client manualny?

**Comment by KillaDreams** [comment]
Yes but things are still broken, ever since it was found out that Eufy was
uploading to the cloud when they said they weren't NOTHING has worked and
nothing has been done to actually fix what's broken, Eufy keeps changing how things work in the background and it's making it hard for @bropat and @fuatakgun to keep up

**Comment by fliespl** [comment]
@KillaDreams got it. But in my case most items work. The only problem was with parsing door sensor states which was fixed in one of newest bropat releases, that's why I am asking if it's possible to update dependency manually to verify :)

**Reply by KillaDreams** [reply]
Unless the door sensor states were fixed in the last day or so they are still not working.  I'm using Bropats version

**Reply by KillaDreams** [reply]
The problem isn't in the addon, it's in the integration, that's what is not receiving the state changes

**Reply by KillaDreams** [reply]
But since you haven't figured it out yet, add-ons, add-on store, repositories, then https://github.com/bropat/hassio-eufy-security-ws

**Reply by fliespl** [reply]
Once I add this - do I still need both?

https://ss.codeone.pl/ss-2023-03-08-14-37-11-1678282631-75Mtlwvs.png

**Reply by KillaDreams** [reply]
No as I'm pretty sure they will conflict with each other

**Reply by fliespl** [reply]
Thanks for that! Will give it a spin later today :)
